### PR TITLE
feat(watch): add debounce to avoid a long wait before plugin is really update, liv-5996

### PR DESCRIPTION
**Pull request by jinevo**

### CODE REVIEW

**Review by Michaelvilleneuve — APPROVED** [review]
👍

File: src/cmds/watch.js [hunk 1]
```diff
@@ -30,10 +30,9 @@ module.exports = function watch() {
   nodeWatch('./', { 
     recursive: true,
     filter: (f, skip) => {
-      if (/\/node_modules/.test(f)) return skip;
-      if (/\/build/.test(f)) return skip;
```
Comment (Michaelvilleneuve) [inline src/cmds/watch.js:34]: Why did you remove the build folder ?

Comment (jinevo) [reply src/cmds/watch.js:34]: I noticed that those filters were not working because they were prefixed by a `/` which was not working for the root folders (`node_modules/`) instead of (`/node_modules/`). Then after this fix, I also noticed that `build` folder should not be filtered since this is there that we output our bundle file.
